Description: Don't build smb.conf.5 manpage
 This is a temporary workaround for a bug in xsltproc, which crashes on some
 architectures when building the smb.conf.5 manpage
Author: Ivo De Decker <ivo.dedecker@ugent.be>
Bug-Debian: http://bugs.debian.org/750593
Forwarded: not-needed

Index: samba/docs-xml/wscript_build
===================================================================
--- samba.orig/docs-xml/wscript_build
+++ samba/docs-xml/wscript_build
@@ -135,7 +135,7 @@ def SMBDOTCONF_MANPAGE(bld, target):
 
 if ('XSLTPROC_MANPAGES' in bld.env and bld.env['XSLTPROC_MANPAGES']):
 
-    SMBDOTCONF_MANPAGE(bld, 'manpages/smb.conf.5')
+    #SMBDOTCONF_MANPAGE(bld, 'manpages/smb.conf.5')
     bld.SAMBAMANPAGES(manpages)
 
     if bld.CONFIG_SET('WITH_PAM_MODULES') and bld.CONFIG_SET('HAVE_PAM_START'):
